catalog/registry/catalog-registry-cswinputtransformer/src/main/java/ddf/catalog/registry/transformer/RegistryTransformer.java

```diff
@@ -88,7 +78,7 @@ public Metacard transform(InputStream inputStream, String id)
 
         metacard.setAttribute(Metacard.METADATA, xml);
         metacard.setTags(Collections.singleton(RegistryObjectMetacardType.REGISTRY_TAG));
-        metacard.setContentTypeName(RegistryServiceMetacardType.SERVICE_REGISTRY_METACARD_TYPE_NAME);
+        metacard.setContentTypeName(RegistryObjectMetacardType.SERVICE_REGISTRY_METACARD_TYPE_NAME);
```

Comment:
If you make the change in the converter to set the content-type then remove this line here.